### PR TITLE
[README] deprecated infrakit

### PR DESCRIPTION
**Pull request by arthurlogilab**
**- What I did**

updated readme

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by justincormack** [comment]
It was actually archived in 2019.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ LinuxKit, a toolkit for building custom minimal, immutable Linux distributions.
 - Designed to create [reproducible builds](./docs/reproducible-builds.md) [WIP]
 - Designed for building and running clustered applications, including but not limited to container orchestration such as Docker or Kubernetes
 - Designed from the experience of building Docker Editions, but redesigned as a general-purpose toolkit
-- Designed to be managed by external tooling, such as [Infrakit](https://github.com/docker/infrakit) or similar tools
+- Designed to be managed by external tooling, such as [Infrakit](https://github.com/docker/infrakit) (renamed to [deploykit](https://github.com/docker/deploykit) which has been archived in 2018) or similar tools
```
Comment (arthurlogilab) [inline README.md]:
```suggestion
- Designed to be managed by external tooling, such as [Infrakit](https://github.com/docker/infrakit) (renamed to [deploykit](https://github.com/docker/deploykit) which has been archived in 2019) or similar tools
```

**Comment by arthurlogilab** [comment]
@rn I can but I don't know how to do it, can you point me in the right direction ?

**Comment by arthurlogilab** [comment]
@rn ok, I did a `git rebase -i arthur/master` and `git commit --amend --signoff`

hope this helps

**Pull request merged**